app/views/catalog/_within_collection_dropdown.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="input-group w-50">
+  <span class="input-group-prepend">
+    <label class="input-group-text" for="within_collection">
+      <%= t('arclight.within_collection_dropdown.label') %>
```

Comment:
@jvine the current label is "Search" and wasn't sure if that is what we would ultimately want here.  We can add screen ready only text so the label is visually "Search" but reads as "Search...something else" if that would be preferable.  Wasn't exactly sure.

Reply:
I think something else already has the label of "Search" - this might be better as "Search within" where "within" is hidden.